bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -22,6 +22,23 @@ spec:
         sleep 1
       done
   containers:
+  - name: cluster-policy-controller
+    image: ${CLUSTER_POLICY_CONTROLLER_IMAGE}
```

Comment:
:woman_shrugging:

Reply:
@soltysh you know how to plumb the imagestream from the payload?  If not, ask @sanchezl to help you get sorted out.